### ISSUE TITLE
missing one positional arg 'sr'

### ISSUE THREAD

**Issue by shipleyxie** [facebookresearch/denoiser#105]
Got Error "missing one positional arg 'sr'" while running on GPU;

https://github.com/facebookresearch/denoiser/blob/606d7695363422c7a474ab5d46f8a4f266b1e343/denoiser/evaluate.py#L69

And change to 
```python
pool.submit(_run_metrics, clean, estimate, args, args.sample_rate))
```
can solve this problem.

**Comment by sangeet2020** [comment]
Yeah, I too found this error and fixed it manually.

**Comment by jeungmin717** [comment]
thx alot!

**Comment by adefossez** [comment]
thanks for reporting, this should be fixed with 7177ed1896
